src/components/groups/v-button-group/v-button-group.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+	<div class="v-button-group"></div>
+</template>
+
+<script lang="ts">
+import { createComponent, computed, ref } from '@vue/composition-api';
+import useSizeClass, { sizeProps } from '@/compositions/size-class';
+
+interface Button {
+	name: string;
+	active: boolean;
+	icon?: string;
+	foregroundColor?: string;
+	backgroundColor?: string;
+	foregroundColorActive?: string;
+	backgroundColorActive?: string;
+}
+
+export default createComponent({
+	model: {
+		prop: 'input',
+		event: 'change'
+	},
+	props: {
+		input: {
+			type: Array as () => Button[],
```

Comment:
```suggestion
			type: Array as PropType<Button[]>,
```

where PropType is imported from @vue/composition-api